helm/defectdojo/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "2.5.0-dev"
 description: A Helm chart for Kubernetes to install DefectDojo
 name: defectdojo
-version: 1.6.23-dev
+version: 1.6.24-dev
```

Comment:
there's no need anymore to bump the version in PRs. The linting will only check when doing a new release and the version will be bumped automatically as part of the new release aftercare.

Reply:
Should I rollback version than?

Reply:
I think it's cleaner to roll back, but up to you :-)